### PR TITLE
[30] Implement portfolio - part 3

### PR DESCRIPTION
**Pull request by alan-hadyk**

### CODE REVIEW

File: src/UI/molecules/__tests__/Hexagon.test.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import { RenderResult } from "@testing-library/react";
+
+import Hexagon from "<molecules>/Hexagon";
+import { HexagonProps } from "<molecules>/__typings__/Hexagon.d.ts";
+
+import renderWithTheme from "<helpers>/tests/renderWithTheme";
+
+describe("molecules / Hexagon", () => {
+  test("should have correct structure when fill is equal pattern", () => {
```
Comment (alan-hadyk) [inline src/UI/molecules/__tests__/Hexagon.test.tsx:10]:
```suggestion
  test("should have correct structure", () => {
```

File: src/UI/molecules/__tests__/Hexagon.test.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      const { Icon } = setup({
+        fill: "solid"
+      });
+
+      expect(Icon.textContent).toEqual("Hexagon.svg");
+    });
+
+    describe("Props", () => {
+      describe("isActive", () => {
+        test("should have true when fill is solid", () => {
```
Comment (alan-hadyk) [inline src/UI/molecules/__tests__/Hexagon.test.tsx:100]: Missing test - `should have false when fill is none`

File: src/UI/molecules/__tests__/Hexagon.test.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      IconContainer
+    } = setup({
+      fill: "pattern"
+    });
+
+    expect(HexagonContainer.children[0]).toEqual(IconContainer);
+    expect(IconContainer.children[0]).toEqual(Icon);
+    expect(HexagonContainer.children[1]).toEqual(HexagonInnerContainer);
+  });
+
```
Comment (alan-hadyk) [inline src/UI/molecules/__tests__/Hexagon.test.tsx:24]: Missing test - `should be able to render children`
